### ISSUE TITLE
The purpose of the function"custom_mse_loss" in train.py

### ISSUE THREAD

**Issue by Vickeyhw** [aigc-apps/VideoX-Fun#275]
I noticed that in evry training code, the mse loss is reimplemented as custom_mse_loss, in which function, where diff.abs() > threshold the loss is not calculated in the final loss. What is the purpose of it?

**Comment by bubbliiiing** [comment]
There were stability issues when model training was first implemented, but they are rarely encountered now, although the function has been retained.
